README.Docker.md

```diff
@@ -0,0 +1,246 @@
+# ClamAV in Docker
+ClamAV can be run within a Docker container. This provides isolation from other
+processes by running it in a containerized environment. If new or unfamiliar
+with docker, containers or cgroups see [docker.com](https://www.docker.com).
+
+
+## Building the ClamAV container
+While it is recommended to pull the container from our
+[docker hub registry](https://hub.docker.com/r/clamav/clamav), some may want
+to build the container locally instead. All that is needed is
+```console
+docker build --tag "MyClamAV" .
```

Comment:
I found that the first part of the tag (before a `:`) has to be lowercase.  `MyClamAV` doesn't work, `myclamav` is fine.  `myclamav:BLAH` is also fine.

Reply:
I'll admit, I wrote this from memory, I never tagged it as such :) I now have the context to your previous remark. I'll use your suggestion with the ticket, as I think those are the most common use cases.